pyfr/plugins/average.py

```diff
@@ -0,0 +1,63 @@
+# -*- coding: utf-8 -*-
+
+from collections import OrderedDict
+
+import numpy as np
+
+from pyfr.plugins.base import BasePlugin
+
+
+class AveragePlugin(BasePlugin):
+    name = 'average'
+    systems = ['*']
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+        self.nsteps = self.cfg.getint(self.cfgsect, 'nsteps')
+        self.writeskip = self.cfg.getint(self.cfgsect, 'writeskip')
+        self.tout = list(args[0].tout)
```

Comment:
If you need to use the arguments declare them.  `args[0]` gives no indication about what the argument is.